src/components/CourseTable/index.jsx

```diff
@@ -178,6 +171,13 @@ class CourseTable extends React.Component {
               placeholder="Search"
             />
           </div>
+          <div className="px-3 float-right" style={{ width: '12%' }}>
```

Comment:
Either add these styles in an `App.scss` as classes and include them here

Reply:
done